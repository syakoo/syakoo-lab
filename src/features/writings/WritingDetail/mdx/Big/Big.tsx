import { bigStyles } from "./Big.css";

type BigProps = {
  children: React.ReactNode;
};

export const Big: React.FC<BigProps> = ({ children }) => {
  return <p className={bigStyles}>{children}</p>;
};
