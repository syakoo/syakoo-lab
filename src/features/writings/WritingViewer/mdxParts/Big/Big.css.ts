import { style } from "@vanilla-extract/css";
import { tokens } from "@/design-system/tokens";

export const bigStyles = style({
  marginBlock: tokens.spaces[300],
  fontSize: tokens.fontSizes[400],
  fontWeight: "bold",
});
