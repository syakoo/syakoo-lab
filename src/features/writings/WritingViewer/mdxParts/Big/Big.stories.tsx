import type { Meta, StoryObj } from "@storybook/react";

import { Big } from ".";

const meta = {
  component: Big,
} satisfies Meta<typeof Big>;

export default meta;

type Story = StoryObj<typeof meta>;

export const Sample: Story = {
  args: {
    children: "Sample",
  },
};
