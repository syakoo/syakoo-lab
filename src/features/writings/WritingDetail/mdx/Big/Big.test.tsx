import { testStories } from "@/test/testStories";

import * as stories from "./Big.stories";

describe("Big", () => {
  testStories(stories);
});
