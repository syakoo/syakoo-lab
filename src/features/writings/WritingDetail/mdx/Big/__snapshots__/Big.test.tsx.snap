// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Big renders Sample 1`] = `
<body>
  <div>
    <p
      class="Big_bigStyles__17823w10"
    >
      Sample
    </p>
  </div>
</body>
`;
